Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/adapter/AdapterDocRetrieveOrchImpl.java

```diff
@@ -73,17 +71,9 @@ public RetrieveDocumentSetResponseType respondingGatewayCrossGatewayRetrieve(Ret
 
     protected RetrieveDocumentSetResponseType callRedactionEngine(RetrieveDocumentSetRequestType retrieveRequest,
```

Comment:
Similar issue with AdapterDocQueryOrchImpl. Remove method callRedationEngine if we don't use or serve no purpose